frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/DiscoverAction.java

```diff
@@ -142,6 +155,7 @@ public void onSuccess() {
       @Override
       @UIEffect
       public void onThrowable(Throwable e) {
+        val branchLayoutException = e.getCause();
```

Comment:
why is it needed now?

Reply:
The compiler complains that code inside WriteAction.run() throws an uncaught exception, so the solution for that is a try-catch with rethrowing it as RuntimeException.